doc/topics/releases/3000.rst

```diff
@@ -528,11 +483,13 @@ the ``beacon_module`` parameter in the beacon configuration.
             /etc/another_file: {}
         - beacon_module: inotify
 
-Module Changes
-==============
+Enhancements to chroot
+======================
 
-- Added new :py:func:`boto_ssm <salt.modules.boto_ssm>` module to set and query
```

Comment:
I think this is in Neon with https://github.com/saltstack/salt/pull/54982

Reply:
Yes - it was just in the wrong place in the releasenotes and duplicated :trollface: 

It's still here in the right place: https://github.com/saltstack/salt/pull/55882/files/28808792d85f88e6a5cf66629c1274ebf95dd037#diff-0ef54636435038ddec39952d7e8fe37bR418

Reply:
Yes - it was just in the wrong place in the releasenotes and duplicated :trollface: 

It's still here in the right place: https://github.com/saltstack/salt/pull/55882/files/28808792d85f88e6a5cf66629c1274ebf95dd037#diff-0ef54636435038ddec39952d7e8fe37bR418